### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UseSpacing doesn't ignore comment lines

### ISSUE THREAD

**Issue by yguedidi** [slevomat/coding-standard#682]
When using comments arround a `use` statement, for example to use `// phpcs:disable`, the following configuration breaks because it takes into account those comment lines.
I think it shouldn't, or at least it should be something configurable.

```xml
<rule ref="SlevomatCodingStandard.Namespaces.UseSpacing">
    <properties>
        <property name="linesCountAfterLastUse" value="1"/>
        <property name="linesCountBeforeFirstUse" value="1"/>
        <property name="linesCountBetweenUseTypes" value="0"/>
    </properties>
</rule>
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/14fbbf5ec49f09bfe2aafb512ad10fcb86cdd088

**Comment by o5** [comment]
Looks like it's broke again.

Running tests on master with this patch:

```patch
diff --git a/tests/Sniffs/Namespaces/UseSpacingSniffTest.php b/tests/Sniffs/Namespaces/UseSpacingSniffTest.php
index dbd7a6f3..43f089b7 100644
--- a/tests/Sniffs/Namespaces/UseSpacingSniffTest.php
+++ b/tests/Sniffs/Namespaces/UseSpacingSniffTest.php
@@ -22,6 +22,7 @@ class UseSpacingSniffTest extends TestCase
        public function testDefaultSettingsNoErrors(): void
        {
                $report = self::checkFile(__DIR__ . '/data/useSpacingWithDefaultSettingsNoErrors.php');
+               var_dump($report->getErrors());
                self::assertNoSniffErrorInFile($report);
        }

diff --git a/tests/Sniffs/Namespaces/data/useSpacingWithDefaultSettingsNoErrors.php b/tests/Sniffs/Namespaces/data/useSpacingWithDefaultSettingsNoErrors.php
index e343849a..01933a1d 100644
--- a/tests/Sniffs/Namespaces/data/useSpacingWithDefaultSettingsNoErrors.php
+++ b/tests/Sniffs/Namespaces/data/useSpacingWithDefaultSettingsNoErrors.php
@@ -1,7 +1,7 @@
 <?php declare(strict_types = 1);

 use DateTimeImmutable;
-use function phpversion;
+use function phpversion; // foo bar
 use const PHP_VERSION_ID;

 ?><?php
diff --git a/tests/bootstrap.php b/tests/bootstrap.php
index d783176b..d5820d32 100644
--- a/tests/bootstrap.php
+++ b/tests/bootstrap.php
@@ -1,6 +1,6 @@
 <?php declare(strict_types = 1);

-error_reporting(E_ALL);
+error_reporting(E_ALL ^ E_DEPRECATED);

 require __DIR__ . '/../vendor/autoload.php';
 require __DIR__ . '/../vendor/squizlabs/php_codesniffer/autoload.php';
```

```
~/src/github/slevomat-coding-standard master*
❯ bin/phpunit --no-coverage -- tests/Sniffs/Namespaces/UseSpacingSniffTest.php
PHPUnit 9.1.4 by Sebastian Bergmann and contributors.

................Farray(1) {
  [5]=>
  array(1) {
    [1]=>
    array(1) {
      [0]=>
      array(5) {
        ["message"]=>
        string(68) "Expected 0 lines between different types of use statement, found -1."
        ["source"]=>
        string(89) "SlevomatCodingStandard.Namespaces.UseSpacing.IncorrectLinesCountBetweenDifferentTypeOfUse"
        ["listener"]=>
        string(56) "SlevomatCodingStandard\Sniffs\Namespaces\UseSpacingSniff"
        ["severity"]=>
        int(5)
        ["fixable"]=>
        bool(true)
      }
    }
  }
}


Time: 00:00.908, Memory: 10.00 MB

There was 1 failure:

1) SlevomatCodingStandard\Sniffs\Namespaces\UseSpacingSniffTest::testDefaultSettingsNoErrors
No errors expected, but 1 errors found.
Failed asserting that an array is empty.

/Users/o5/src/github/slevomat-coding-standard/SlevomatCodingStandard/Sniffs/TestCase.php:75
/Users/o5/src/github/slevomat-coding-standard/tests/Sniffs/Namespaces/UseSpacingSniffTest.php:26

FAILURES!
Tests: 17, Assertions: 93, Failures: 1.
```

```
PHP 7.4.6
PHP_CodeSniffer 3.5.5
```

**Comment by kukulich** [comment]
@o5 Please open new issue.
